iroh-net/src/magicsock/endpoint.rs

```diff
@@ -44,6 +44,15 @@ const UPGRADE_INTERVAL: Duration = Duration::from_secs(60);
 /// How long we trust a UDP address as the exclusive path (without using DERP) without having heard a Pong reply.
 const TRUST_UDP_ADDR_DURATION: Duration = Duration::from_millis(6500);
 
+/// Number of addresses that are not active that we keep around per peer.
+///
+/// See [`Endpoint::prune_direct_addresses`].
+const MAX_INACTIVE_DIRECT_ADDRESSES: usize = 5;
+
+/// Number of peers that are inactive for which we keep info about. This limit is enforced
+/// periodically via [`PeerMap::prune_inactive`].
+const MAX_INACTIVE_PEERS: usize = 30;
```

Comment:
I wonder if we should make this number configurable, large nodes might want to keep more than this around